### ISSUE TITLE
Tracking Issue for AArch64 SHA3 intrinsics

### ISSUE THREAD

**Issue by Amanieu** [rust-lang/rust#117225]
Feature gate: `#![feature(stdarch_neon_sha3)]`

This is a tracking issue for NEON intrinsics under the `sha3` feature.

### Public API

```rust
// core::arch::aarch64

fn veor3q*(..);
fn vbcaxq*(..);
fn vrax1q*(..);
fn vsha512*(..);
```

### Steps / History

- [ ] Implementation: #...
- [ ] Final comment period (FCP)[^1]
- [ ] Stabilization PR

### Unresolved Questions

- None yet.

[^1]: https://std-dev-guide.rust-lang.org/feature-lifecycle/stabilization.html

**Comment by Amanieu** [comment]
Yes, just submit a stabilization PR and then we can start an FCP.

**Comment by tarcieri** [comment]
It looks like since it's autogenerated code, stabilizing `vsha512*` would involve stabilizing all of `stdarch_neon_sha3` by modifying this line, which I can open a PR for: https://github.com/rust-lang/stdarch/blob/3961d3b/crates/stdarch-gen-arm/src/main.rs#L532

**Comment by tarcieri** [comment]
Opened a PR: https://github.com/rust-lang/stdarch/pull/1552

**Comment by Jamesbarford** [comment]
Triaging

Can we close this ticket since; https://github.com/rust-lang/stdarch/pull/1552 has been merged?
